ui/js/graphs.js

```diff
@@ -738,8 +754,53 @@ perf.controller('TestChooserCtrl', function($scope, $modalInstance, $http,
         $scope.selectedProject = projects[0];
     }
     $scope.loadingTestData = false;
-
     var series = [];
+    // if we choose to add all related platforms, firstly we need
+    // to obtain all the platform of this project, then pick out the
+    // specific test by platform and project.
+    if(option === "addRelatedPlatforms") {
```

Comment:
If you follow my suggestion above, this will become

    if (options.addRelatedPlatforms)

Be careful about spacing. There should be one between 'if' and '(options...'.